### PR TITLE
proposal to add EPAM company

### PR DESCRIPTION
**Pull request by ykachube**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @ykachube, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit fe10876).  See below for details:

```
epam.md: Invalid section: "Engineering the Future".  Expected one of: ["Company blurb","Company size","Remote status","Region","Company technologies","Office locations","How to apply"]
epam.md: Required section "Company blurb" not found.

2 problems detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @ykachube, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 4cd1ca5). :heavy_check_mark:

File: README.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ Name | Website | Region
 [EngineYard (Support Team)](/company-profiles/engineyard.md) ⚠️️ | https://www.engineyard.com/ |
 [Enjoei](/company-profiles/enjoei.md) ⚠️️ | https://www.enjoei.com.br/ | Brazil
 [Envato](/company-profiles/envato.md) | https://envato.com/ | Worldwide
+[Epam](/company-profiles/epam.md) | https://epam.com/ | Belarus, Russia and Ukraine.
```
Comment (dougaitken) [inline README.md]: Region is shown as "Belarus, Russia and Ukraine." but the site linked in the company profile mentioned "Anywhere"

https://anywhere.epam.com/

If so, please change this to `Worldwide`

File: company-profiles/epam.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Remote status
+
+Partially, details available on subdomain https://anywhere.epam.com/
+Work and Learn
+Anywhere with EPAM
+After 25+ years of leading the world in product engineering, we’re bringing our experience, training and global customers to you. Anywhere.
+
+## Region
+
+Russia, Belarus and Ukraine
```
Comment (dougaitken) [inline company-profiles/epam.md]: Same as comment left on main readme - 



Region is shown as "Belarus, Russia and Ukraine." but the site linked mentioned "Anywhere" - https://anywhere.epam.com/

If so, please change this to `Worldwide` - region is "Where this company employees are located/can be hired in"

File: README.md (hunk 1 above)
Comment (ykachube) [reply README.md]: ok, makes sense to me, thank you

**Comment by dougaitken** [comment]
Thanks @ykachube!

**Pull request merged**